athena-gcs/src/main/java/com/amazonaws/athena/connectors/gcs/glue/PartitionResolver.java

```diff
@@ -0,0 +1,35 @@
+/*-
+ * #%L
+ * athena-gcs
+ * %%
+ * Copyright (C) 2019 - 2022 Amazon Web Services
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+package com.amazonaws.athena.connectors.gcs.glue;
+
+import com.amazonaws.athena.connectors.gcs.common.PartitionResult;
+import com.amazonaws.services.glue.model.Table;
+import org.apache.arrow.vector.complex.reader.FieldReader;
+
+import java.util.Map;
+
+public interface PartitionResolver
```

Comment:
Delete this interface (entire file) since there's only a single concrete implementation and that concrete implementation is being constructed and used directly by the code anyway.

Reply:
Removed.